Assets/BattleUIController.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Newtonsoft.Json.Bson;
+using TMPro;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+public class BattleUIController : MonoBehaviour
+{
+    [SerializeField]
+    public Button[] optionButtons;
+    [SerializeField]
+    public ScrollRect itemList;
+    
+    public CombatMenu mainCombatMenu;
+    public CombatMenu secondCombatMenu;
+    public CombatMenu itemCombatMenu;
+
+    public void LoadAndRenderMenu(CombatMenu menu)
+    {
+        UnbindListeners();
+        for (int i=0; i < menu.menuNames.Length; i++)
```

Comment:
What kind of formatting do we plan to use for these files?

Reply:
you mean code style? anything for now. I'm not too particular in code style for a project like this.